.ci/Jenkinsfile

```
@@ -51,7 +51,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger("${obltGitHubComments()}")
+    issueCommentTrigger("${obltGitHubComments()}|^/test benchmark fullreport")
```

to support the existing default commands and the new one 
```suggestion
    issueCommentTrigger("(${obltGitHubComments()}|^/test benchmark fullreport)")
```